### ISSUE TITLE
Labels for Statusmappings in Administration missing in Shopware 6.3.5.2

### ISSUE THREAD

**Issue by FloKnapp** [PAYONE-GmbH/shopware-6#89]
Hi,

i recently integrated PayOne into Shopware 6.3.5.2. Unfortunately the Labels within the Statusmapping-Section aren't visible.

I was able to pin it down to the `payone-settings`-Snippet from `./Resources/app/administration/module/page/payone-settings.html.twig:136`:

```
<sw-form-field-renderer
    v-if="element.name.startsWith('PayonePayment.settings.paymentStatus')"
    :config="{
        componentName: 'sw-single-select',
        label: getInlineSnippet(element.config.label),
        helpText: getInlineSnippet(element.config.helpText),
        options: stateMachineTransitionActions,
    }"
    v-model="config[element.name]"
/>
```

The part where the label is set doesn't seem to work. The options were generated though.

Do you have any ideas how this could be solved?

Thank you in advance!

**Comment by azngeek** [comment]
Is there an update on this issue?

**Comment by MarkusSchmalhoferSonoportal** [comment]
We have the same problem.

**Comment by hreinberger** [comment]
This has been fixed in 2.2.0-beta.1: https://github.com/PAYONE-GmbH/shopware-6/releases/tag/v2.2.0-beta.1

**Comment by fgiering** [comment]
The fix is just for Shopware 6.4. right? I do not see any difference in 6.3.5.4
